contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSelectionList.vue

```diff
@@ -137,4 +137,9 @@
     }
   }
 
+  .list-card-hover {
+    margin: 16px;
+    box-shadow: 0 3px 5px 0 rgba(0, 0, 0, 0.2);
```

Comment:
I think if we want the cards to have a box shadow when unhovered, we should just remove the `flat` prop.